newpipe.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+export JAVA_HOME=/app/share/java_home
+export LD_LIBRARY_PATH=/app/lib/art
+ln -s $XDG_CACHE_HOME ~/.cache
```

Comment:
this `~/.cache` won't persist with your current finish args but not sure if it needs to. Why does it not use XDG_CACHE_HOME directly?

Reply:
The symlink is not preserved, but the content lands in $XDG_CACHE_HOME, which is preserved. I have also fixed it upstream here: https://gitlab.com/android_translation_layer/art_standalone/-/commit/ad8a1a0e40e2156b9e62f588f9927f6148c2acc7, but it is not in the release yet.